lib/assets/components/explore_tiles.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:app/models/Action.dart';
+import 'package:app/models/Campaign.dart';
+import 'package:app/models/Cause.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class ExploreCampaignTile extends StatelessWidget {
+  final String headerImage;
+  final String title;
+  final ListCause? cause;
+
+  ExploreCampaignTile(ListCampaign model, {Key? key})
+      : headerImage = model.headerImage,
+        title = model.title,
+        cause = model.cause,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(8)),
+      clipBehavior: Clip.antiAlias,
+      child: AspectRatio(
+        aspectRatio: 0.75,
+        child: Column(
+          children: [
+            AspectRatio(
+              aspectRatio: 1.5,
+              child: Image.network(
+                headerImage,
+                fit: BoxFit.cover,
+              ),
+            ),
+            Expanded(
+              child: Container(
+                alignment: Alignment.centerLeft,
+                padding: const EdgeInsets.symmetric(horizontal: 12),
+                child: _ExploreTileTitle(title),
+              ),
+            ),
+            Padding(
+                padding: const EdgeInsets.only(bottom: 8),
+                child: _ExploreTileCause(cause!)),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class ExploreActionTile extends StatelessWidget {
+  final String title;
+  final ActionType type;
+  final Color iconColor;
+  final Color headerColor;
+  final Color dividerColor;
+  final IconData icon;
+  final ListCause? cause;
```

Comment:
```suggestion
  final ListCause cause;
```

Reply:
Hey @JElgar , I think you might've been looking at an older commit of this pr